metricbeat/module/kubernetes/event/event.go

```diff
@@ -97,7 +95,6 @@ func generateMapStrFromEvent(eve *Event) common.MapStr {
 	eventMeta := common.MapStr{
 		"timestamp": common.MapStr{
 			"created": eve.Metadata.CreationTimestamp,
-			"deleted": eve.Metadata.DeletionTimestamp,
```

Comment:
Does this mean we no longer store `deleted` timestamp? it should be removed from fields.yml if that's the case

Reply:
based on https://github.com/vjsamuel/beats/blob/a22d999d3f076d7e9fda6cf93e7a883acbf02913/metricbeat/module/kubernetes/event/event.go#L87 we never write into the destination backend events that are deleted as it would skew event counts. hence it doesn't make sense to even have a deleted timestamp field in both the event and the fields.yml. So i have gone ahead and removed it.